proto/internalapi/scanner/v4/vulnerability_report.proto

```diff
@@ -53,6 +59,7 @@ message VulnerabilityReport {
     string fixed_in_version = 11;
     CVSS cvss = 12 [deprecated = true]; // value of cvss field is included in cvss_metrics field, the exact deprecation date is undecided
     repeated CVSS cvss_metrics = 13;
+    EPSS epss = 14;
```

Comment:
Sanity check: We don't foresee "Multiple EPSS scores" the way we do for CVSS, right?

Reply:
Also, if possible, I would like to call this `epss_metrics`, and be aligned with `cvss_metrics`.

Reply:
> We don't foresee "Multiple EPSS scores" the way we do for CVSS, right?
Yeah, it's one EPSS score per CVE as far as I know

> I would like to call this epss_metric, and be aligned with cvss_metrics
How about `epss_detail`? Because it has score date, model version, etc, not everything is numeric.

Reply:
I think `epss` is fine as-is. `cvss_metrics` made sense since metrics is a term I've seen a bunch in the specs (though our use here may be wrong...)